tee-worker/common/core-primitives/node-api/api-client-types/src/lib.rs

```diff
@@ -74,6 +79,105 @@ pub type ParentchainSignature = Signature<ParentchainSignedExtra>;
 /// Signature type of the [UncheckedExtrinsicV4].
 pub type Signature<SignedExtra> = Option<(Address, PairSignature, SignedExtra)>;
 
+// The following types are copied from the substrate-api-client crate to be able to encode/decode them
+/// Simplified TransactionStatus to allow the user to choose until when to watch
+/// an extrinsic.
+// Indexes must match the substrate_api_client::TransactionStatus::as_u8
```

Comment:
Can we have a test case so we can prove they match ?

Reply:
done